src/sentry/snuba/metrics/utils.py

```diff
@@ -158,6 +159,7 @@
     "generic_metrics_counters": OP_TO_SNUBA_FUNCTION["metrics_counters"],
     "generic_metrics_distributions": OP_TO_SNUBA_FUNCTION["metrics_distributions"],
     "generic_metrics_sets": OP_TO_SNUBA_FUNCTION["metrics_sets"],
+    "generic_metrics_gauges": {},
```

Comment:
Ideally we undo changes to this and similar files that aren't directly needed for metric extraction from alerts.